sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
+       =======
+       Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Value
```

Comment:
```suggestion
       x0   : Integer
```
Integer might be better than `Value`

Reply:
Well, it can take up decimal values too.

Reply:
Then, maybe try something like `Number`.
`Value` doesn't  exactly explain a type imo.

Reply:
Yes, I thought about it too. Alright, I am changing this to ``Number``